README.md

```diff
@@ -241,12 +242,55 @@ stores.forEach { store ->
     store.initialize()
 }
 
-// Optional: add logging interceptor to log action events
-dispatcher.addInterceptor(LoggerInterceptor(stores)) { tag, msg ->
+// Optional: add logging middleware to log action events
+dispatcher.addMiddleware(LoggerMiddleware(stores)) { tag, msg ->
     Log.d(tag, msg)
 }
 ```
 
+### \[Android] Kodein Android utils
+`mini-kodein-android` has some utils methods in order to inject an Android's `ViewModel` in a `DIAware` `Activity` or `Fragment`.
+To use it, bind with kodein the Android's `ViewModelProvider.Factory` instance:
+```kotlin
+// Use any tag to differ between the injected `Context` or `Application` if you are binding also `Context` with Kodein
+bind<Application>("appTag") with singleton { app }
+bind<ViewModelProvider.Factory>() with singleton { DIViewModelFactory(di.direct) }
+```
+To inject a view model without parameters, bind it as follows in this example:
+```kotlin
+bindViewModel { MainViewModel(instance("appTag") }
+```
+And in your `DIAware` `Activity` or `Fragment`:
+```kotlin
+private val mainViewModel: MainViewModel by viewModel()
+```
+
+`mini-kodein-android-compose` has some utils methods in order to inject an Android's `ViewModel` in the scope of a Navigation component
+graph. This is useful as in Jetpack Compose is common to have only one or few `Activities` and none `Fragment` so, in order to scope
+the lifecycle of the `ViewModel` not for all the life of the `Activity`, we can scope it to any route existing in the `NavBackStackEntry`.
+For use it, do the same as above, but instead of injecting the view model scoped to a route of the Navigation, the `NavHost` composable
```

Comment:
```suggestion
In order to use it, do the same as above, but instead of injecting the ViewModel scoped to a route of the Navigation, the `NavHost` composable
```